### PR TITLE
Harrison/docs update comm

### PR DESCRIPTION
**Pull request by hwchase17**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -330,15 +331,55 @@ what you wanted by clicking the `View deployment` or `Visit Preview` buttons on
 This will take you to a preview of the documentation changes.
 This preview is created by [Vercel](https://vercel.com/docs/getting-started-with-vercel).
 
-## 🏭 Release Process
+## 📕 Releases & Versioning
 
 As of now, LangChain has an ad hoc release process: releases are cut with high frequency by
-a developer and published to [PyPI](https://pypi.org/project/langchain/).
+a developer and published to [PyPI](https://pypi.org/project/langchain/).
```
Comment (nfcampos) [inline .github/CONTRIBUTING.md:337]: Shouldn't this be pointing to pypi pages for each package?

File: README.md [hunk 2]
```diff
@@ -45,7 +45,10 @@ This framework consists of several parts.
 - **[LangServe](https://github.com/langchain-ai/langserve)**: A library for deploying LangChain chains as a REST API.
 - **[LangSmith](https://smith.langchain.com)**: A developer platform that lets you debug, test, evaluate, and monitor chains built on any LLM framework and seamlessly integrates with LangChain.
 
-**This repo contains the `langchain` ([here](libs/langchain)), `langchain-experimental` ([here](libs/experimental)), and `langchain-cli` ([here](libs/cli)) Python packages, as well as [LangChain Templates](templates).**
+The LangChain libraries themselves are made up of several different packages.
+- **[`langchain-core`](libs/core)**: Base abstractions and LangChain Expression Language.
+- **[`langchain-community`](libs/community)**: Third party integrations.
+- **[`langchain`](libs/langchain)**: Chains, agents, and retrieval strategies that make up an application's cognitive architecture.
```
Comment (nfcampos) [inline README.md:51]: This one doesn't mention experimental, on purpose?

File: docs/docs/get_started/introduction.mdx [hunk 3]
```diff
@@ -29,6 +29,11 @@ The main value props of the LangChain packages are:
 
 Off-the-shelf chains make it easy to get started. Components make it easy to customize existing chains and build new ones.
 
+The LangChain libraries themselves are made up of several different packages.
+- **`langchain-core`**: Base abstractions and LangChain Expression Language.
+- **`langchain-community`**: Third party integrations.
+- **`langchain`**: Chains, agents, and retrieval strategies that make up an application's cognitive architecture.
```
Comment (nfcampos) [inline docs/docs/get_started/introduction.mdx:35]: same as above

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -70,17 +70,18 @@ Install Poetry: **[documentation on how to install it](https://python-poetry.org
...
-### Core vs. Experimental
+### Different projects
 
-This repository contains three separate projects:
-- `langchain`: core langchain code, abstractions, and use cases.
-- `langchain_core`: contain interfaces for key abstractions as well as logic for combining them in chains (LCEL).
-- `langchain_experimental`: see the [Experimental README](https://github.com/langchain-ai/langchain/tree/master/libs/experimental/README.md) for more information.
+This repository contains multiple projects:
+- `langchain_core`: Base interfaces for key abstractions as well as logic for combining them in chains (LangChain Expression Language).
+- `langchain-community`: Third-party integrations and implementations of various components.
```
Comment (baskaryan) [inline .github/CONTRIBUTING.md:77]: langchain_community

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -70,17 +70,18 @@ Install Poetry: **[documentation on how to install it](https://python-poetry.org
...
 tell Poetry to use the virtualenv python environment (`poetry config virtualenvs.prefer-active-python true`)
 
-### Core vs. Experimental
+### Different projects
 
-This repository contains three separate projects:
-- `langchain`: core langchain code, abstractions, and use cases.
-- `langchain_core`: contain interfaces for key abstractions as well as logic for combining them in chains (LCEL).
-- `langchain_experimental`: see the [Experimental README](https://github.com/langchain-ai/langchain/tree/master/libs/experimental/README.md) for more information.
+This repository contains multiple projects:
```
Comment (baskaryan) [inline .github/CONTRIBUTING.md:75]: should just say packages instead of projects to be unambiguous

Comment (baskaryan) [reply .github/CONTRIBUTING.md:75]: same in header above

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -70,17 +70,18 @@ Install Poetry: **[documentation on how to install it](https://python-poetry.org
...
+- `langchain_core`: Base interfaces for key abstractions as well as logic for combining them in chains (LangChain Expression Language).
+- `langchain-community`: Third-party integrations and implementations of various components.
+- `langchain`: Chains, agents, and retrieval logic that makes up the cognitive architecture of your applications.
+- `langchain_experimental`: Components or chains that are more experimental in nature or unsafe (contain CVEs).
```
Comment (baskaryan) [inline .github/CONTRIBUTING.md:79]: maybe something like:

Components and chains that are experimental, either in the sense that the techniques are novel and still being tested, or they require giving the LLM more access than would be possible in most production systems.

File: .github/CONTRIBUTING.md (hunk 4 above)
Comment (baskaryan) [reply .github/CONTRIBUTING.md:77]: Could just say "third-party integrations"

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (baskaryan) [reply .github/CONTRIBUTING.md:337]: cut with high frequency by a *maintainer*

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -330,15 +331,55 @@ what you wanted by clicking the `View deployment` or `Visit Preview` buttons on
...
+- Bug fixes
+- New features
+- Any changes to private interfaces
+- Any changes to `langchain_core.beta`
+
+### `langchain`
+
+`langchain` is currently on version `0.0.x`
+
+All changes will be accompanied by a patch version increase. Any changes to public interfaces are nearly always done in a backwards compatible way.
```
Comment (baskaryan) [inline .github/CONTRIBUTING.md:361]: worth saying "nearly always done in a backwards compatible way and will be communicated ahead of time when they are not backwards compatible"?

File: README.md (hunk 2 above)
Comment (baskaryan) [reply README.md:51]: worth mentioning cli?

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -330,15 +331,55 @@ what you wanted by clicking the `View deployment` or `Visit Preview` buttons on
...
+All changes will be accompanied by a patch version increase.
+
+### `langchain-experimental`
+
+`langchain-experimental` is currently on version `0.0.x`
+
+All changes will be accompanied by a patch version increase.
+
+
+## 🏭 Release Process
```
Comment (baskaryan) [inline .github/CONTRIBUTING.md:378]: delete

**Review by baskaryan — COMMENTED** [review]
there's a docs/docs/get_started/installation page we should probably update

File: README.md (hunk 2 above)
Comment (hwchase17) [reply README.md:51]: somewhat on purpose not mentioning both

**Pull request merged**